docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx

```diff
@@ -32,31 +32,36 @@ Implementing a custom form element in VertiGIS Studio Mobile consists of three s
 
 ## Set up the Custom Form Element Skeleton
 
-1. Create a new file `LoadingIndicator.cs` in the platform agnostic project of the VertiGIS Studio Mobile [Quickstart](../mobile/sdk-overview.mdx).
-2. Add a new skeleton form element that implements `FormComponent`.
+1. Using Visual Studio, create a new file, using the Xamarin Forms: Content View template. Name the file `LoadingIndicator.xaml`, and place it in the platform agnostic project of the VertiGIS Studio Mobile [Quickstart](../mobile/sdk-overview.mdx).
+2. Change the base class to `ContentComponent` both in the xaml and in the code-behind.
+
+```xml title="App1/App1/workflow/LoadingIndicator.xaml"
+<?xml version="1.0" encoding="UTF-8"?>
+// highlight-next-line
```

Comment:
I don't think this is doing what you intended it to. For me, I just see the comment text appear as-is, it doesn't actually highlight the line in the rendered Markdown.

Same thing goes for the other usages later in the file.

Reply:
Looks correct to me. How are you viewing the file? Are you viewing the 'built' app, or just the raw mdx file?

![image](https://user-images.githubusercontent.com/55208417/176001741-3e01207b-f7b3-4e00-b519-2c85598ae998.png)

Reply:
Oh, my bad. I was viewing the Markdown as rendered by Github.